net/znc/Makefile

```diff
@@ -277,7 +278,7 @@ CONFIGURE_VARS += \
 	LIBS="-lstdc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
 
 CONFIGURE_ARGS += \
-	--disable-charset \
+	$(if $(CONFIG_ZNC_ICU), --enable-charset, --disable-charset))
```

Comment:
Trust no one. ;-) People do make mistakes or might misremember stuff (I'm quite good at this). Also it's good practice to ensure stuff is still working after changes, regardless how sure you are about the changes.